example/spec/standalone/redeployment/LambdaVersionOption.js

```diff
@@ -1,85 +0,0 @@
-'use strict';
-
-const fs = require('fs-extra');
-const jsyaml = require('js-yaml');
-
-const { getJsonS3Object } = require('@cumulus/aws-client/S3');
-const { cf } = require('@cumulus/aws-client/services');
-const StepFunctions = require('@cumulus/aws-client/StepFunctions');
-const { getWorkflowFileKey } = require('@cumulus/common/workflows');
-
-const {
-  loadConfig,
-  runKes
-} = require('../../helpers/testUtils');
-
-describe('When the useWorkflowLambdaVersions option is set to false the deployment', () => {
```

Comment:
I would prefer `xdescribe`ing this test, with a `TODO: reimplement for Terraform` note.